lib/cfa/augeas_parser.rb

```diff
@@ -19,6 +36,7 @@ def add(value, placer = AppendPlacer.new)
       element = placer.new_element(@tree)
       element[:key] = augeas_name
       element[:value] = value
+      # FIXME: load_collection missing here
```

Comment:
It is actually used and tested in sudoers which I use as initial testing lense and used also in some test in rspec.